Formula/volta.rb

```diff
@@ -0,0 +1,27 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta.git", :revision => "10a9bee148d2a833069d6cc27aea56af3092dc6e"
+  version "0.7.2"
+
+  depends_on "rust" => :build
+
+  conflicts_with "hadoop", :because => "both install `yarn` binaries"
+  conflicts_with "node", :because => "both install `node` binaries"
+  conflicts_with "npm", :because => "both install `npm` binaries"
+  conflicts_with "yarn", :because => "both install `yarn` binaries"
+
+  def install
+    ENV["VOLTA_HOME"] = prefix.to_s
+    system "./dev/unix/volta-install.sh", "--release", "--skip-setup"
+  end
+
+  def postinstall
+    chmod "+x", "#{opt_prefix}/bin/volta"
```

Comment:
This should be fixed upstream, it's failing the build.

```
 ==> brew audit volta --online --new-formula
Error: 1 problem in 1 formula detected
volta:
  * Non-executables were installed to "/usr/local/opt/volta/bin"
    The offending files are:
      /usr/local/opt/volta/bin/volta
```

Reply:
I'll raise a ticket upstream then! Thanks!

Reply:
Please link the upstream ticket in the formula.